### ISSUE TITLE
Unable to start app

### ISSUE THREAD

**Issue by fmkizz** [glushchenko/fsnotes#1776]
### Description

For some time now, and after updating the app a couple of times, there is this problem that keeps coming back. After tapping the icon the startup screen is displayed for about 10 seconds and then it dissappears without starting the app. It's a bit unpredictable: sometimes the app will function for a while, but all of a sudden the problem returns. It looks like toggling between icloud and local storage sometimes resolves it, but sometimes not. At this moment the app is not working. IOS 17.6.1 on SE 2020

### To Reproduce

1 Start app by tapping icon
2 Watch startup screen for 10 sec
3 App crashes

### Expected behavior

I expect the app to startup normally off course.

### FSNotes version

6.9.4

### macOS/iOS version

IOS 17.6.1

**Comment by glushchenko** [comment]
If the application does not start, you may have put too many files in the folder to be scanned. Try moving unused files from Files.app to another directory and start the application.

**Comment by fmkizz** [comment]
Hi, I removed a reasonable amount of files from the home directory. After that I managed to start the app, but run into the same problem later on. There is one folder containing about 1200 entries, is that too much for the app to handle maybe? I didn't have this problem until a few months ago.. And might it help if I subdivide this folder into smaller portions? Thnx

**Comment by fmkizz** [comment]
Hi, any more reflections on this? I really like this app, but I also really like to have this amount of files (fyi: total file size only about 3MB) to my disposal. Is there any way this app will be able to handle this? Thnx

**Comment by glushchenko** [comment]
I have over a thousand notes, everything works fine. I can offer access to beta version in testflight, but I need your email.

**Comment by fmkizz** [comment]
Ok, worth giving that a try, thnx. How do I provide you with my address, besides posting it here?

**Comment by glushchenko** [comment]
Just write me fluder@icloud.com or in telegram @hlushchenko

**Comment by glushchenko** [comment]
@fmkizz ?

**Comment by fmkizz** [comment]
Hi, if you mean you're waiting for my response: I sent you an email to the mentioned address about 2 hours ago..

**Comment by glushchenko** [comment]
@fmkizz done. Check TestFlight.

**Comment by glushchenko** [comment]
@fmkizz did you test it?

**Comment by fmkizz** [comment]
Yes, it doesn't crash during startup, but it takes forever to load sometimes. Sometimes loading goes quickly, a bit unpredictable.. Until yesterday that is, because it looks like today this behaviour is over. Checked a couple of times and the app starts up / loads without delay.

**Comment by glushchenko** [comment]
I'll say it again, a programme can take a long time to start if there are a lot of files in the file system. I recommend checking for hidden folders and dot folders in the directory where notes are stored.

**Comment by fmkizz** [comment]
Hi, the issue seemed to have dissappeared a couple of days ago, but later on it returned. When I switched to the non-testflight version I also faced the crashing issue again. I don't have any hidden files in the fsnotes directory, only the files I mentioned before: around 1200 (total size 3MB), just a bit more than the number of files you say you have. Off course I can try to reduce the number of files, but that would make this app less useful / attractive to me..
